huntsman/utils/light_control.py

```diff
@@ -0,0 +1,265 @@
+import qhue
+from os import path
+from qhue import Bridge, QhueException, create_new_username
+
+#note, light 5 can be 'uncommented' once installed, or causes problems using currently 
+
+
+#deal with error system - should errors be sent to pocs? how does pocs handle error messages?
+
+def manual_login_phillips_hue(hue_ip="10.88.21.10", verbose = False):
+    
+    """
+    A way to manually login to the hue bridge, requires the button on the bridge to be pushed.
+    Only useful should auto login fail and present at the dome.
+    
+    Inputs:
+        
+        hue_ip - the ip address of the hue bridge. This is is set to the default input "10.88.21.10" 
+    
+    """
+    
+    username = qhue.create_new_username(hue_ip)
+    
+    if verbose:
+    
+       print("The username created is", username)
+    
+    b = qhue.Bridge(hue_ip,username)
+    
+    return(b)
+    
+def automatic_login_phillips_hue(hue_ip = "10.88.21.10", file_path = "hue_username.txt", verbose = False):
```

Comment:
We'll need the yaml config here too.

Reply:
Is there a yaml file for this I can play with or shall I leave the yaml config until it's been finalised?

Reply:
You will want to create something like `conf_files/lights.yaml`. Then you can do:

```
from pocs.utils.config import load_config

config_fn = '{}/conf_files/{}'.format(os.getenv('HUNTSMAN_POCS'), 'lights.yaml')
config = load_config(config_files=[config_fn])
```

This assumes you have the $HUNTSMAN_POCS environment variable set.

This also is obviously just using a separate config file from the others, which is fine for now. We might combine them or keep it separate.